tensorflow/contrib/rnn/python/ops/core_rnn_cell_impl.py

```diff
@@ -83,12 +83,14 @@ def call(self, inputs, state):
 class GRUCell(RNNCell):
   """Gated Recurrent Unit cell (cf. http://arxiv.org/abs/1406.1078)."""
 
-  def __init__(self, num_units, input_size=None, activation=tanh, reuse=None):
+  def __init__(self, num_units, input_size=None, activation=tanh, reuse=None,
+               initializer=None):
     super(GRUCell, self).__init__(_reuse=reuse)
     if input_size is not None:
       logging.warn("%s: The input_size parameter is deprecated.", self)
     self._num_units = num_units
     self._activation = activation
+    self._initializer = initializer
```

Comment:
@lukaszkaiser @fchollet @ebrevdo 

Yeap, I'm checking the keras documents for this issue.

To the point, I have a question.
Is 'kernel_initializer' and 'bias_initializer' a string parameter like 'zeros' 'ones' 
or it can just directly insert the parameters into initial values?

I found some strings for initializer like this

'ones'
'zeros'
'random_uniform'
'glorot_normal'
https://keras.io/initializers/#usage-of-initializers

but in tensorflow, the initializer only directly assigned the initializer parameters to the initial values.